src/validate.ts

```diff
@@ -131,14 +131,27 @@ export const checkEnvironmentVariables = (): void => {
       'Environment variable SENTRY_ORG is missing an organization slug'
     );
   }
-  if (!process.env['SENTRY_PROJECT']) {
+  if (!process.env['SENTRY_AUTH_TOKEN']) {
     throw Error(
-      'Environment variable SENTRY_PROJECT is missing a project slug'
+      'Environment variable SENTRY_AUTH_TOKEN is missing an auth token'
     );
   }
-  if (!process.env['SENTRY_AUTH_TOKEN']) {
+};
+
+export const getProjects = (): string[] => {
+  const projectsOption = core.getInput('projects') || '';
+  const projects = projectsOption
+    .split(' ')
+    .map(proj => proj.trim())
+    .filter(proj => !!proj);
```

Comment:
Should we also `uniq` the list?

Reply:
@mgaeta I don't think that's necessary. The user should know better than to add the same project multiple times. We can let the CLI handle that case however it does.